wiki/models/custom-sabers.md

```diff
@@ -1,13 +1,13 @@
 # Custom Sabers
 ## Installation
-The latest Custom Saber plugin can be found in the [mod installers](/beginners-guide.md#installers). Run ModAssistant, select the Custom Saber plugin, and click Install.
-Once you've installed it you will see a folder called `CustomSabers` in [your install folder](/faq/install-folder.md), this is where you should place the `*.saber` files you want to use. You can download more sabers on [ModelSaber](https://modelsaber.com/Sabers/). There is a limit to the number of sabers that varies by machine, but the average number of sabers supported seems to be around 10.
+The latest Custom Saber plugin can be found in [Mod Assistant](https://github.com/Assistant/ModAssistant/releases/latest). Run ModAssistant, select the Custom Saber plugin, and click Install.
```

Comment:
Mod Assistant only applies to users playing on PC. I think it'd be better to include some instructions for Quest players as well